rllib/examples/rl_modules/classes/autoregressive_actions_rlm.py

```diff
@@ -0,0 +1,295 @@
+from abc import abstractmethod
+from typing import Any, Dict, Type
+
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.rllib.core import Columns
+from ray.rllib.core.models.base import ENCODER_OUT
+from ray.rllib.core.models.catalog import Catalog
+from ray.rllib.core.models.configs import MLPHeadConfig
+from ray.rllib.core.models.specs.specs_dict import SpecDict
+from ray.rllib.core.rl_module.rl_module import RLModule, SingleAgentRLModuleSpec
+from ray.rllib.core.rl_module.torch.torch_rl_module import TorchRLModule
+from ray.rllib.examples.envs.classes.correlated_actions_env import CorrelatedActionsEnv
+from ray.rllib.models.distributions import Distribution
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.torch_utils import convert_to_torch_tensor
+from ray.rllib.utils.typing import TensorType
+
+from ray.tune import register_env
+
+register_env("correlated_actions_env", lambda _: CorrelatedActionsEnv(_))
+
+
+torch, nn = try_import_torch()
+
+
+class AutoregressiveActionRLM(RLModule):
```

Comment:
Let's add a docstring here explaining how to use this class.

Which algos would it work with? PPO only?
What else is required to make this work? Any other config settings?